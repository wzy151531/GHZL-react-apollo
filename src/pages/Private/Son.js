import React, { PureComponent } from 'react';

export default class Son extends PureComponent {
  render() {
    console.log('Son PureComponent render');
    return (
      <div>{this.props.sonMsg}</div>
    );
  }
}