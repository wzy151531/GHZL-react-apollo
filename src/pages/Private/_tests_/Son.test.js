import React from 'react';
import Son from '../Son';
import renderer from 'react-test-renderer';

test('Son should render correctly', () => {
  const component = renderer.create(
    <Son sonMsg="sonMsg"/>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});