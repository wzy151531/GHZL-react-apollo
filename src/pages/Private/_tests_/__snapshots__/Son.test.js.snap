// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Son should render correctly 1`] = `
<div>
  sonMsg
</div>
`;
